### PR TITLE
upgrade to Provenance v1.19.1 and Provwasm v2.4.0

### LINKED ISSUE #10: Update to Provenance 1.19
## Summary

provwasm-test-tube needs to be updated to the latest version of provenance

## Proposal

update to the latest provenance v1.19 version and the latest provwasm with grpc updates

____

#### For Admin Use

- [ ] Not duplicate issue
- [ ] Appropriate labels applied
- [ ] Appropriate contributors tagged
- [ ] Contributor assigned/self-assigned

### PR DESCRIPTION
**Pull request by kwtalley**
This PR updates to the latest Provenance v1.19.1 and Provwasm v2.4.0. This includes the changes from sdk 0.50 and migration to grpc message types. All supported module endpoints are updated and cargo manifests have been refactored to use workspace management

closes #10

### CODE REVIEW

File: packages/provwasm-test-tube/src/module/msgfees.rs [hunk 1]
```diff
@@ -23,15 +26,31 @@ where
         pub assess_custom_msg_fee: MsgAssessCustomMsgFeeRequest["/provenance.msgfees.v1.MsgAssessCustomMsgFeeRequest"] => MsgAssessCustomMsgFeeResponse
     }
 
-    fn_query! {
-        pub query_params ["/provenance.msgfees.v1.Query/Params"]: QueryParamsRequest => QueryParamsResponse
+    fn_execute! {
```
Comment (iramiller) [inline packages/provwasm-test-tube/src/module/msgfees.rs:29]: none of these tx are something that a person/contract can execute directly ... they would need a gov proposal to change these settings within the msg fee module.

Comment (kwtalley) [reply packages/provwasm-test-tube/src/module/msgfees.rs:29]: The queries as well?

Comment (iramiller) [reply packages/provwasm-test-tube/src/module/msgfees.rs:29]: Queries are fine... it is just that the msg fee settings are governance controlled.  A user might want to know what fees are configured though so the queries make sense.

Comment (kwtalley) [reply packages/provwasm-test-tube/src/module/msgfees.rs:29]: fixed

**Review by iramiller — APPROVED** [review]

**Pull request merged**